### DISCUSSION TITLE
Per-world Tablist

### DISCUSSION THREAD

**Discussion by Ericwtwpower** [Zrips/CMI#8079] [Feature Requests]
Make General Tablist like this: https://github.com/NEZNAMY/TAB/wiki/Feature-guide:-Header-&-Footer#per-world--per-server

**Comment by CloudeLecaw** [comment]
You can make allready have each world having its own header and footer by world. with CMIs tablist

Creat different Tablists in the grouped once
and give permsision for the correct tablist in each world with your permission system.
Your permissionsystem like Pex and LP can give permissions allready

Open Tablist.yml

```
  GroupFormat:
    '1':
      PlayerName: '[playerDisplayName]r'
      Header:
        '1':
        - 'Header 1'
      Footer:
        '1':
        - 'Footer 1'
    '2':
      PlayerName: '[playerDisplayName]'
      Header:
        '1':
        - 'Header 2'
      Footer:
        '1':
        - 'Footer 2'
    '3':
      PlayerName: '[playerDisplayName]'
      Header:
        '1':
        - 'Header 3'
      Footer:
        '1':
        - 'Footer 3'
 ```
Now give permissions `cmi.tablist.1` in world1 `cmi.tablist.2` in world2 and `cmi.tablist.3` in world3
make sure that OPs, Owner and Admins have the right negations in the right world to not see only tablist3
